aggregator/aggregator.go

```diff
@@ -22,21 +22,32 @@ import (
 	oprsinfoserv "github.com/Layr-Labs/eigensdk-go/services/operatorsinfo"
 )
 
+// The aggregator is the entity responsible of communicating with the BLS aggregation service. This includes:
+//   - Listening to new task created events, initializing new tasks at the BLS aggregation service for them.
+//   - Receiving responses to tasks from the operators, and sending them to the BLS aggregation service.
+//   - Receiving the aggregated responses from the BLS aggregation service, and sending them to the Task Manager
+//
+// Most of these things are delegated to the Task Processor interface, that process tasks and communicates with
+// the on-chain task manager contract.
```

Comment:
The BLS aggregation service is an implementation detail. We should avoid mentioning that in the user documentation.

```suggestion
// The aggregator is responsible for aggregating signed task responses from operators and posting them on chain. This includes:
//   - Listening to new task created events.
//   - Receiving signed responses from the operators.
//   - Sending the aggregated responses to the `TaskManager` contract
//
// Most of these things are delegated to the `TaskProcessor` interface, that processes
// tasks and communicates with the on-chain `TaskManager` contract.
```